keras_cv/layers/preprocessing3d/frustum_random_point_feature_noise_test.py

```diff
@@ -0,0 +1,74 @@
+# Copyright 2022 The KerasCV Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+import numpy as np
+import tensorflow as tf
+
+from keras_cv.layers.preprocessing3d import base_augmentation_layer_3d
+from keras_cv.layers.preprocessing3d.frustum_random_point_feature_noise import (
+    FrustumRandomPointFeatureNoise,
+)
+
+POINT_CLOUDS = base_augmentation_layer_3d.POINT_CLOUDS
+BOUNDING_BOXES = base_augmentation_layer_3d.BOUNDING_BOXES
+POINTCLOUD_LABEL_INDEX = base_augmentation_layer_3d.POINTCLOUD_LABEL_INDEX
+
+
+class FrustumRandomPointFeatureNoiseTest(tf.test.TestCase):
+    def test_augment_point_clouds_and_bounding_boxes(self):
```

Comment:
For this test, could we specify a set of e.g. 10 known points where half are inside and half are outside of the frustum, and then assert that exactly the half that we expect to have noise added are the ones that have noise added?

Reply:
Add a new test `test_augment_specific_point_clouds_and_bounding_boxes`